pylint_odoo/checkers/no_modules.py

```diff
@@ -109,6 +109,14 @@
         'manifest-author-string',
         settings.DESC_DFLT
     ),
+    'E%d02' % settings.BASE_NOMODULE_ID: (
```

Comment:
Could you change the number to last... E.g. 99?
This is a fork... Then to avoid future conflicts with new checks from original project